src/styles/variables/variables-z-index.less

```diff
@@ -18,7 +18,7 @@
 @z-index-header-border: 1;
 @z-index-header-tabs: @z-index-header-border + 1;
 
-@z-index-page-body: 1;
+@z-index-page-body: 0;
```

Comment:
Why do we need this change?

Reply:
because ui-kit does not have a default z-Index and thus the new modal would vanish behind the page-body for no good reason. it does not seem sensible to set a z-index of `1` on a normal page.